creator-node/package.json

```diff
@@ -19,7 +19,7 @@
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
-    "@audius/libs": "1.2.93",
+    "@audius/libs": "1.2.102",
```

Comment:
just a reminder that when changing libs version in CN or other service, we need to be mindful of unintended regressions

Reply:
thanks, I just confirmed this now by looking at the diff between those versions and re-running maddog. will follow up with the maddog results. the most relevant changes between these versions seem to be:
- https://github.com/AudiusProject/audius-protocol/commit/2ad6dc613beb3ffb69c5c3cdd66e2a30c53f09d9
- https://github.com/AudiusProject/audius-protocol/commit/00b5bb5a429fc732f62261a43cfa27151db925fe

Reply:
sweet, both should be no-ops